packages/components/src/draggable/index.native.js

```diff
@@ -63,7 +63,7 @@ const Draggable = ( {
 	useAnimatedReaction(
 		() => isDragging.value,
 		( result, previous ) => {
-			if ( result === previous || previous === null ) {
+			if ( result === previous ) {
```

Comment:
Noting that we need to manually test the drag-and-drop features to ensure this does not cause regressions.